CONTRIBUTING.md

```diff
@@ -17,10 +17,10 @@ When you make a pull request it will be reviewed. You should also update the `CH
 ## Making a new release
 
 1. Create a new branch
-2. Update the version number in `plugin.php` and `package.json` to reflect the nature of the changes, this plugin follows semver versioning. 
+2. Update the version number in `plugin.php` and `package.json` to reflect the nature of the changes, this plugin follows semver versioning.
   - For small changes like bug fixes update the patch version
   - For changes that add functionality without changing existing functionality update the minor version
   - For breaking or highly significant changes update the major version
 3. Add a title heading for the version number above the latest updates in `CHANGELOG.md`
 4. Create a pull request for the branch
-5. Once merged a release will be built and deployed by CircleCI corresponding to the version number in `package.json`
+5. Once merged go to the releases tab and create a new release, set the new tag name to match the updated version number
```

Comment:
Should this step say to create the new release from the master branch? Not 100% clear how to manage backports or what branch we should be releaseing from, from this wording

Reply:
Oh no, you'd create it from the relevant version branch. I think the release process is a bit more complicated now tbh. One moment.